src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertTo-Html.cs

```diff
@@ -198,6 +199,27 @@ public SwitchParameter Fragment
 
         private SwitchParameter _fragment;
 
+        /// <summary>
+        /// Gets or sets the _hyperlink switch, if true, then auto
+        /// generate hyperlinks for URI.
+        /// </summary>
+        [Parameter(ParameterSetName = "Hyperlink")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter Hyperlink
```

Comment:
Would it be better to call this `-AutoHyperlink`?

Reply:
I agree! I have changed the parameter to autohyperlink